src/types/components/accordions/accordions.interface.ts

```diff
@@ -0,0 +1,13 @@
+import { SxProps } from "@mui/material"
+
+export interface AccordionItem {
+    title: string
+    description: string
+}
+interface NestedStyles {
+    [key: string]: string | number | NestedStyles
+}
+export interface AccordionSx {
+    withIcon: { [key: string]: SxProps | NestedStyles }
+    noIcon: { [key: string]: SxProps | NestedStyles }

```

Comment:
do noIcon?: , so it will be not required, and you wont have to pass empty object

Reply:
does it have a sense to type styles if field is not required and those who reuse it is choosing what to style just by checking component

Reply:
yes it have, styles as other props have to be typed

Reply:
ok